.circleci/config.yml

```diff
@@ -344,6 +344,7 @@ workflows:
             branches:
               ignore:
                 - master
+                - '^snyk-fix.*'
```

Comment:
Does this carat indicate "snyk-fix" should appear at the start of the branch name? 
I cannot find any instances of this in the documentation: [https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpull_requestpull_request_targetbranchesbranches-ignore](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpull_requestpull_request_targetbranchesbranches-ignore)